src/app/core/admin/user-management/admin-list-users.component.ts

```diff
@@ -150,15 +155,18 @@ export class AdminListUsersComponent extends AbstractPageableDataComponent<User>
 			});
 
 		this.sortEvent$.next(this.headers.find((header: any) => header.default) as SortChange);
-
-		super.ngOnInit();
 	}
 
 	confirmDeleteUser(user: User) {
 		const id = user.userModel._id;
-		const username = user.userModel.username;
-
-		console.error('Delete User not yet implemented.');
+		this.adminUsersService.removeUser(id).subscribe({
+			next: data => {
+				this.reload();
+			},
+			error: error => {
+				console.error('There was an error!', error);
```

Comment:
@stella-masucci - since we have an alert service available in this class, we can let the users know whether there was a failure using something like `this.alertService.addClientErrorAlert(error);`